Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -559,24 +561,25 @@ namespace O3DE::ProjectManager
             }
 
             auto result = m_register.attr("register")(
-                QString_To_Py_Path(engineInfo.m_path),
-                pybind11::none(), // project_path
-                pybind11::none(), // gem_path
-                pybind11::none(), // external_subdir_path
-                pybind11::none(), // template_path
-                pybind11::none(), // restricted_path
-                pybind11::none(), // repo_uri
-                pybind11::none(), // default_engines_folder
-                QString_To_Py_Path(engineInfo.m_defaultProjectsFolder),
-                QString_To_Py_Path(engineInfo.m_defaultGemsFolder),
-                QString_To_Py_Path(engineInfo.m_defaultTemplatesFolder),
-                pybind11::none(), // default_restricted_folder
-                QString_To_Py_Path(engineInfo.m_thirdPartyPath),
-                pybind11::none(), // external_subdir_engine_path
-                pybind11::none(), // external_subdir_project_path
-                false, // remove
-                force
-            );
+                "engine_path"_a                  = QString_To_Py_Path(engineInfo.m_path),
+                "project_path"_a                 = pybind11::none(),
+                "gem_path"_a                     = pybind11::none(),
+                "external_subdir_path"_a         = pybind11::none(),
+                "template_path"_a                = pybind11::none(),
+                "restricted_path"_a              = pybind11::none(),
+                "repo_uri"_a                     = pybind11::none(),
+                "default_engines_folder"_a       = pybind11::none(),
+                "default_projects_folder"_a      = QString_To_Py_Path(engineInfo.m_defaultProjectsFolder),
+                "default_gems_folder"_a          = QString_To_Py_Path(engineInfo.m_defaultGemsFolder),
+                "default_templates_folder"_a     = QString_To_Py_Path(engineInfo.m_defaultTemplatesFolder),
+                "default_restricted_folder"_a    = pybind11::none(),
+                "default_third_party_folder"_a   = QString_To_Py_Path(engineInfo.m_thirdPartyPath),
+                "external_subdir_engine_path"_a  = pybind11::none(),
+                "external_subdir_project_path"_a = pybind11::none(),
+                "external_subdir_gem_path"_a     = pybind11::none(),
+                "remove"_a                       = false,
+                "force"_a                        = force
```

Comment:
not necessary for this review, but for the future - the beauty of using the literals is you can simplify this to remove all the default none arguments
```suggestion
                "engine_path"_a                  = QString_To_Py_Path(engineInfo.m_path),
                "default_projects_folder"_a      = QString_To_Py_Path(engineInfo.m_defaultProjectsFolder),
                "default_gems_folder"_a          = QString_To_Py_Path(engineInfo.m_defaultGemsFolder),
                "default_templates_folder"_a     = QString_To_Py_Path(engineInfo.m_defaultTemplatesFolder),
                "default_third_party_folder"_a   = QString_To_Py_Path(engineInfo.m_thirdPartyPath),
                "remove"_a                       = false,
                "force"_a                        = force
```

For now let's go with what you have so because re-testing all this code will take some time and we can combine these changes with other upcoming PRs that touch this code